ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -818,6 +819,17 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                                 gru_seq->get_linear_before_reset());
             break;
         }
+        case OP_TYPEID::HardSigmoid:
```

Comment:
I think that ref impl should be called as evaluate method in ngraph/core/src/op/hard_sigmoid